eth/backend.go

```diff
@@ -193,6 +193,7 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 		chainConfig.CancunTime = config.OverridePassedForkTime
 		chainConfig.HaberTime = config.OverridePassedForkTime
 		chainConfig.HaberFixTime = config.OverridePassedForkTime
+		chainConfig.PascalTime = config.OverridePassedForkTime
```

Comment:
OverridePassedForkTime is invented for reth, and use in dev or QA env
because geth client accept `0` as timestamp, but reth not
so set hard fork timestamps those passed on mainnet using this flag,
so `chainConfig.PascalTime = config.OverridePassedForkTime` is not suitable here

I think you can add OverridePascal, following OverrideBohr